paris/src/main/java/com/airbnb/paris/typed_array_wrappers/TypedArrayWrapper.kt

```diff
@@ -29,6 +30,7 @@ abstract class TypedArrayWrapper {
             R.array.null_,
             R.color.null_,
             R.drawable.null_,
+            R.font.null_,
```

Comment:
👍 👍 I'll have to remember to edit https://github.com/airbnb/paris/wiki/View-Recycling#null-values